src/controllers/question.js

```diff
@@ -0,0 +1,198 @@
+import Questions from '../models/Question';
+import User from '../models/User';
+//import Bid from '../models/Bid';
+
+import mongoose from 'mongoose';
+const getAllQuestions = async (req, res) => {
+  try {
+    const questions = await Questions.find().populate('user', {
+      firstName: 1,
+      lastName: 1,
+      profilePhoto: 1
+    });
+    return res.status(200).json({
+      message: 'List Question',
+      data: questions,
+      error: false
+    });
+  } catch (error) {
+    return res.status(500).json({
+      message: `An error ocurred: ${error}`,
+      data: undefined,
+      error: true
+    });
+  }
+};
+
+const getQuestionById = async (req, res) => {
+  try {
+    const { id } = req.params;
+    const idQuestion = await Questions.findOne({ $and: [{ _id: id }] }).populate('user', {
+      firstName: 1,
+      lastName: 1,
+      profilePhoto: 1
+    });
+    if (!idQuestion) {
+      return res.status(404).json({
+        message: `Question with ID ${id} was not found`,
+        data: undefined,
+        error: true
+      });
+    }
+    return res.status(200).json({
+      message: `Question with ID ${idQuestion.id} was found!`,
```

Comment:
Here the ID is enough to show

```suggestion
      message: `Question with ID ${id} was found!`,
```